core/src/test/java/com/linecorp/armeria/client/RestClientTest.java

```diff
@@ -109,7 +109,7 @@ void restApi(RestClient restClient) {
                     preparation = restClient.delete("/rest/{id}");
                     break;
             }
-            assert preparation != null;
+            assertThat(preparation).isNotNull();
```

Comment:
Thanks for cleaning this up, @ikhoon!

Reply:
New [Error Prone ](https://errorprone.info/bugpattern/UseCorrectAssertInTests) does not allow `assert` in tests.